.github/workflows/release-30_publish_release_draft.yml

```diff
@@ -80,7 +79,6 @@ jobs:
           ./scripts/release/build-changelogs.sh
 
           echo "Checking the folder state"
```

Comment:
very nitpick but: `echo "Checking the folder state: $PWD"` instead of a seperate pwd?

Reply:
I think I'll delete this one and the line below. It was anyway done more for debug purpose.